ebean-core/src/main/java/io/ebeaninternal/server/type/ScalarTypeJsonList.java

```diff
@@ -68,14 +68,24 @@ public JsonB(DocPropertyType docType, boolean nullable) {
    */
   @SuppressWarnings("rawtypes")
   private abstract static class Base extends ScalarTypeJsonCollection<List> {
+    private final boolean keepSource;
 
-    public Base(int dbType, DocPropertyType docType, boolean nullable) {
+    public Base(int dbType, DocPropertyType docType, boolean nullable, boolean keepSource) {
       super(List.class, dbType, docType, nullable);
+      this.keepSource = keepSource;
+    }
+    
+    @Override
+    public boolean isJsonMapper() {
```

Comment:
checkme: Should we refactor that getter to `isKeepSource` `pushSource` / `popSource`